static/src/javascripts/bootstraps/enhanced/minute.js

```diff
@@ -0,0 +1,4 @@
+define([], function () {
+    window._min =  {_publisher: 'MIN-21000'};
```

Comment:
Yup - it is unsafe.
We have access to their developers, so I will ask them to have some form of constructor that can take this.